clang/include/clang/Basic/Builtins.td

```diff
@@ -203,6 +203,12 @@ def FmaF16F128 : Builtin, F16F128MathTemplate {
   let Prototype = "T(T, T, T)";
 }
 
+def Fmaf16 : Builtin, F16F128MathTemplate {
```

Comment:
Why is this needed when we have: https://github.com/llvm/llvm-project/blob/a3913a19185e3513a317aab792e85d5c50ef0242/clang/include/clang/Basic/Builtins.td#L201

That seems to already support fp16 for fma?